sale_configurator_pricelist_tax/models/sale_order.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    price_tax_state = fields.Selection(related="order_id.price_tax_state")
```

Comment:
@sebastienbeau , I don't find the definition of price_tax_state in sale.order dependencies.

Reply:
My bad, i find it here : https://github.com/akretion/sale-workflow/pull/27